### PR TITLE
adopt public resource

### PR DESCRIPTION
**Pull request by ce1adon**
Update repo address

### CODE REVIEW

**Review by pfultz2 — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -192,11 +192,11 @@ pipeline {
             description: "")
         booleanParam(
             name: "MIOPENTENSILE",
-            defaultValue: false,
+            defaultValue: true,
```
Comment (atamazov) [inline Jenkinsfile:195]: Please revert this

File: Jenkinsfile [hunk 2]
```diff
@@ -192,11 +192,11 @@ pipeline {
...
             description: "")
         booleanParam(
             name: "MIOPENTENSILE_LATEST",
-            defaultValue: false,
+            defaultValue: true,
```
Comment (atamazov) [inline Jenkinsfile:199]: And this.

**Review by atamazov — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (ce1adon) [reply Jenkinsfile:195]: Will do when CI passes.

**Comment by atamazov** [comment]
There is no need to change Jenkinsfile to just to toggle some stage ON or OFF.

BTW I just re-run the CI job with "MIOPENTENSILE" and "SMOKE_MIOPENTENSILE_LATEST" manually set. This should be enough to test this PR.

**Review by shurale-nkn — DISMISSED** [review]
But in future, please use Jenkins GUI for parameters changing and start testing job manually.

**Comment by ce1adon** [comment]
> But in future, please use Jenkins GUI for parameters changing and start testing job manually.
> 

Isn't this going to make the tracking of the test results more complex? How do you distinguish the parameter you toggled for each test for the same PR?

**Comment by ce1adon** [comment]
> There is no need to change Jenkinsfile just to toggle some stage ON or OFF.
> 
> BTW I just re-run the CI job with "MIOPENTENSILE" and "SMOKE_MIOPENTENSILE_LATEST" manually set (the rest is cleared). This should be enough to test this PR.

Please also check with Paul.

**Comment by shurale-nkn** [comment]
> 
> Isn't this going to make the tracking of the test results more complex? How do you distinguish the parameter you toggled for each test for the same PR?

The time required to start testing is incomparably less than the duration of testing. It also saves a lot more time than you spent. We hope that all participants are able to visually compare the list of running stages and determine whether they are enough or not. 
Jenkins keeps all job logs for analysis.

**Comment by ce1adon** [comment]
@shurale-nkn Looks like jenkins is still down.

**Comment by atamazov** [comment]
> @shurale-nkn Looks like jenkins is still down.

There were transient problems. It works somehow; I just restarted the job.

**Comment by junliume** [comment]
> > @shurale-nkn Looks like jenkins is still down.
> 
> There were transient problems. It works somehow; I just restarted the job.

Is the Jenkins issue still persistent? I think devops upgraded and downgraded it today due to plugin compatibility issue

**Comment by ce1adon** [comment]
> > > @shurale-nkn Looks like jenkins is still down.
> > 
> > 
> > There were transient problems. It works somehow; I just restarted the job.
> 
> Is the Jenkins issue still persistent? I think devops upgraded and downgraded it today due to plugin compatibility issue

In this PR ,the Jenkins skips a lot of test stages. Not sure if this is related to the early jenkins issue.

**Comment by atamazov** [comment]
> In this PR ,the Jenkins skips a lot of test stages. Not sure if this is related to the early jenkins issue.

That was done intentionally. I unchecked the most of build parameters (manually, in the Jenkins Web UI).

**Comment by atamazov** [comment]
@ce1adon Please resolve https://github.com/ROCmSoftwarePlatform/MIOpen/pull/868#pullrequestreview-639129688 and I will merge it.

**Comment by ce1adon** [comment]
> @ce1adon Please resolve [#868 (review)](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/868#pullrequestreview-639129688) and I will merge it.

I don't see param control lists on jenkins. Are all test stages disabled in default?

**Comment by atamazov** [comment]
> 
> 
> > @ce1adon Please resolve [#868 (review)](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/868#pullrequestreview-639129688) and I will merge it.
> 
> I don't see param control lists on jenkins. Are all test stages disabled in default?

## At http://micimaster.amd.com/job/MLLibs/job/MIOpen, find your branch and click it.

image

## Click on "Build with parameters"

image

## Set build parameters as you wish and click [Build]
image

**Comment by atamazov** [comment]
@ce1adon When you update and push the branch, you may find that "**_Build with parameters_**" is not available, only "**_Build now_**".

In this case you need to give some time for Jenkins to detect and launch CI job (or better click "**_Scan repository now_**" and it will scan immediately). Then check that the job is actually launched and refresh the web page. You'll see that "**_Build now_**" is changed to "**_Build with parameters_**". Then terminate the just launched job and follow the instructions shown [above](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/868#issuecomment-823529901).

**Comment by ce1adon** [comment]
@atamazov 
Is the [skip CI] commit key word still valid?

**Comment by atamazov** [comment]
It should, but please use lowercase.

**Comment by ce1adon** [comment]
This PR passed all the tests. Let me know if you request any change. I will revert the build param after addressing all comments.

**Comment by atamazov** [comment]
@ce1adon The only leftover is https://github.com/ROCmSoftwarePlatform/MIOpen/pull/868#pullrequestreview-639129688.

**Comment by ce1adon** [comment]
> @ce1adon The only leftover is [#868 (review)](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/868#pullrequestreview-639129688).

Please find latest commit for it.

**Review by atamazov — APPROVED** [review]

**Pull request merged**